target/linux/realtek/files-5.10/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -1356,6 +1342,9 @@ static void rtnc_839x_hw_en_rxtx(struct rtnc_priv *priv)
 	/* set and enforce maximum RX len */
 	sw_w32((RTNC_MAX_RX_LEN << 5) | 0x10, priv->r->dma_if_ctrl);
 
+	/* Disable Head of Line feature for all RX rings */
+	sw_w32(0xffffffff, RTL839X_DMA_IF_RX_RING_CNTR);
```

Comment:
curious this one is, you copy pasted the section from `rtl838x_hw_en_rxt` including the comment. That's fine, but then the content, you changed `priv->r->dma_if_rx_ring_size(0)` to `RTL839X_DMA_IF_RX_RING_CNTR` Why not size? why not `priv->r->dma_if_rx_ring_cntr(0)`? why is rtl839x different in this regard?